### DISCUSSION TITLE
BRouter no-go areas

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#479] [Q&A] [answered]
Setting Nogo Aerias (e.g. environmental zones in France) in the BRouter is a bit complicated.
->first save a point in the favorites (nogo(radius in m)name),
then export (nogo folder), then import again.
Couldn't this be made easier somehow?

**Comment by devemux86** [comment]
Route planning with no-go areas will be included inside Cruiser in the near future. :slightly_smiling_face:

**Comment by devemux86** [comment, marked as answer]
Currently you can open gpx files with no-go areas in BRouter advanced settings.

An example by @WolfgangZ42: [nogo_Munich_MR.gpx.txt](https://github.com/devemux86/cruiser/files/14882099/nogo_Munich_MR.gpx.txt)

Instructions:
- https://github.com/devemux86/cruiser/discussions/85#discussioncomment-6091177
